.local.env

```diff
@@ -10,3 +10,7 @@ NX_API_ROOT_URL=http://localhost:3333
 NX_API_METADATA_URL=http://164.92.196.139:3001
 GOOGLE_CLIENT_ID=599384179420-k6tsfpgst2fsgp39iq8cdcb15pdrokih.apps.googleusercontent.com
 GOOGLE_CLIENT_SECRET=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
+NX_API_SPACES=https://mimir-content.fra1.digitaloceanspaces.com
+SPACE_ENDPOINT=fra1.digitaloceanspaces.com 
+SPACE_ID=DO00PBQ8YCVQPFPVD8EP
+SPACE_SECRET=j4jgGNBloFWMRxbMkqauC/xUvNhLgezQwqPnuHoLG6M
```

Comment:
Thanks for exposing sensitive private data to the Internet :D

Reply:
@sabrina34352 can you hide the data from `.local.env` and create some sort of config file that export the data from `.env` and if there is no data return mocked creds (like `space-id: process.env["SPACE-ID"] || "space-id"` as well for other fields)? We don't wanna expose this data, just put it in your `.env` that is protected from git indexing if you need it